ffs/scheduler/scheduler.go

```diff
@@ -396,28 +459,29 @@ func (s *Scheduler) executeQueuedStorage(j ffs.StorageJob) {
 	cancelChan := make(chan struct{}, 1)
 	// Create chan to allow Job cancellation.
 	s.cancelLock.Lock()
-	s.cancelChans[j.ID] = cancelChan
+	s.jobsCancel[j.ID] = cancelChan
 	s.cancelLock.Unlock()
 	defer func() {
 		s.cancelLock.Lock()
-		delete(s.cancelChans, j.ID)
+		delete(s.jobsCancel, j.ID)
 		s.cancelLock.Unlock()
 	}()
 
 	ctx, cancel := context.WithCancel(context.WithValue(context.Background(), ffs.CtxKeyJid, j.ID))
 	defer cancel()
 	ctx = context.WithValue(ctx, ffs.CtxStorageCid, j.Cid)
+	ctx = context.WithValue(ctx, ffs.CtxAPIID, j.APIID)
```

Comment:
We add `APIID` to the ctx that is used by Jobs, so all messages logged are stored for the corresponding FFS instance.